### ISSUE TITLE
Supermodel config files not found

### ISSUE THREAD

**Issue by Ardend** [trzy/Supermodel#88]
I know Supermodel is in progress, but just to experiment I tried adding it manually via custom systems to call the Linux version EmulationStation suggests using here: https://gitlab.com/es-de/emulationstation-de/-/blob/master/USERGUIDE.md#arcade-and-neo-geo

When I set up the files, regardless of where I move the config and NVRAM folders, I get an error that I believe is supposed to say "Unable to open 'Config/Supermodel.ini'. Configuration will not be loaded", only the part in single quotes is garbled.  I also tried the windows version in custom systems and I get the same error, only its not garbled.  And I cant get it to find the Config folder no matter where I put it.  Yet it finds the folder just fine when it goes to write to the error log.  EmulationStations documentation says that Supermodel has to be installed in the ~/Applications directory, but for the deck it has to be in /home/deck/Applications and I am wondering if that is the issue?  Wouldn't the custom systems xml override that?

**Comment by trzy** [comment]
You need to find out what Supermodel's current working directory is. I don't know how EmuStation launches apps. It is entirely possible that they place the .exe in one directory but then launch it with the current working directory somewhere else. If you can find Supermodel.log afterwards and inspect it, it may give you a clue as to where the app was started from. The Config folder would need to be placed there. Currently, you can point Supermodel at a different location for Games.xml but not Supermodel.ini itself.

**Comment by mdeguzis** [comment]
Use the CLI, it should end up trying to use ~/.config/supermode/Config or something. I copied the Config files from this repo after making the dir and it seemed to list the available game compatibility fine.

**Comment by mdeguzis** [comment]
Yes, the flatpak needs to include these files, but you can fix this on your own for now:
```
$ flatpak run com.supermodel3.Supermodel -print-games
Supermodel: A Sega Model 3 Arcade Emulator (Version 0.3a-WIP)
Copyright 2003-2023 by The Supermodel Team
Error: Unable to open '/home/deck/.config/supermodel/Config/Supermodel.ini'. Configuration will not be loaded.
Error: Failed to parse /home/deck/.config/supermodel/Config/Games.xml (XML_ERROR_FILE_NOT_FOUND).
```

```
mkdir ~/src
git clone https://github.com/trzy/Supermodel/ ~/src/supermodel
mkdir -p ~/.config/supermodel/Config

$ cp -v ~/src/supermodel/Config/* ~/.config/supermodel/Config/
'/home/deck/src/supermodel/Config/Games.xml' -> '/home/deck/.config/supermodel/Config/Games.xml'
'/home/deck/src/supermodel/Config/Supermodel.ini' -> '/home/deck/.config/supermodel/Config/Supermodel.ini'
```

Now this should work fine:
```
flatpak run com.supermodel3.Supermodel -print-games

```

**Comment by mdeguzis** [comment]
I lied, I did need to set the starting directory when using the flatpak, but this was true of when I built the Linux binary as well.
